chapter04/qli/ex3.pl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+sub total {
+  my $total = 0;
+  foreach my $num (@_){
+    $total += $num;
+  }
+  return $total;
+}
+
+sub average {
+  if( @_ <= 0 ){
+    return 0;
+  }
```

Comment:
Nice way to prevent dividing by 0. Why do you have the "<" though? Can you enter -1 arguments?